src/foundry/client/pixi/extensions/graphics.d.mts

```diff
@@ -0,0 +1,35 @@
+export {};
+
+type GraphicsPath = [number[] | PIXI.IPointData[] | PIXI.Polygon] | Array<number[] | PIXI.IPointData>;
```

Comment:
This one's on me but on closer inspection it should be `Array<number | PIXI.IPointData>`. I just wasn't thinking when I translated it over.